neurots/generate/diametrizer.py

```diff
@@ -397,7 +407,9 @@ def build(
     and should only update the neuron object.
     """
     if neurite_types is None:
-        neurite_types = [STR_TO_TYPES.get(tree_type) for tree_type in ["apical", "basal"]]
+        neurite_types = [
+            getattr(SectionType, tree_type) for tree_type in ["apical_dendrite", "basal_dendrite"]
+        ]
```

Comment:
```suggestion
        neurite_types = [SectionType.apical_dendrite, SectionType.basal_dendrite]
```
Isn't it possible to just write this?